try:
    number = 3
    print(number)
except:
    print("An error has occurred")


num1 = 45
num2 = 0
try:
  print(num / num2)
except:
    print("An error has occurred")
finally:
    print("Success")
